doc/releases/changelog-0.11.0.md

```diff
@@ -202,79 +265,83 @@
+    llvm.func @teardown() {
+      llvm.call @__catalyst__rt__finalize() : () -> ()
+      llvm.return
+    }
+  }
+  ```
 
-<h3>Deprecations 👋</h3>
+* The error messages that indicate invalid `scale_factors` in `catalyst.mitigate_with_zne` have been
+  improved to be formatted properly.
+  [(#1603)](https://github.com/PennyLaneAI/catalyst/pull/1603)
 
 <h3>Bug fixes 🐛</h3>
 
-* Fixed `argnums` parameter of `grad` and `value_and_grad` being ignored.
+* Fixed the `argnums` parameter of `grad` and `value_and_grad` being ignored.
   [(#1478)](https://github.com/PennyLaneAI/catalyst/pull/1478)
 
-* All dialects are loaded preemptively.
-  This allows third-party plugins to load their dialects.
+* All dialects are loaded preemptively. This allows third-party plugins to load their dialects.
   [(#1584)](https://github.com/PennyLaneAI/catalyst/pull/1584)
 
-* Fixed an issue ([(#1488)](https://github.com/PennyLaneAI/catalyst/pull/1488)) where Catalyst could
-  give incorrect results for circuits containing `qml.StatePrep`.
+* Fixed an issue where Catalyst could give incorrect results for circuits containing `qml.StatePrep`.
   [(#1491)](https://github.com/PennyLaneAI/catalyst/pull/1491)
 
-* Fixes an issue ([(#1501)](https://github.com/PennyLaneAI/catalyst/issues/1501)) where using
-  autograph in conjunction with catalyst passes causes a crash.
+* Fixed an issue where using autograph in conjunction with catalyst passes caused a crash.
   [(#1541)](https://github.com/PennyLaneAI/catalyst/pull/1541)
 
-* Fixes an issue ([(#1548)](https://github.com/PennyLaneAI/catalyst/issues/1548)) where using
-  autograph in conjunction with catalyst pipeline causes a crash.
+* Fixed an issue where using
+  autograph in conjunction with catalyst pipeline caused a crash.
   [(#1576)](https://github.com/PennyLaneAI/catalyst/pull/1576)
 
-* Fixes an issue ([(#1547)](https://github.com/PennyLaneAI/catalyst/issues/1547)) where using
-  chained catalyst passes decorators causes a crash.
+* Fixed an issue where using
+  chained catalyst passes decorators caused a crash.
   [(#1576)](https://github.com/PennyLaneAI/catalyst/pull/1576)
 
-* Autographs and catalyst pipeline fix.
+* Specialized handling for `pipeline`s was added.
   [(#1599)](https://github.com/PennyLaneAI/catalyst/pull/1599)
 
-* Fixes an issue ([(#1339)](https://github.com/PennyLaneAI/catalyst/issues/1339)) where using
-  autograph with control/adjoint functions used on operator objects causes a crash.
+* Fixed an issue where using
+  autograph with control/adjoint functions used on operator objects caused a crash.
   [(#1605)](https://github.com/PennyLaneAI/catalyst/pull/1605)
 
-* Fixes an issue ([(#896)](https://github.com/PennyLaneAI/catalyst/issues/896)) where using
-  pytrees inside a loop with autograph causes falling back to python.
+* Fixed an issue where using
+  pytrees inside a loop with autograph caused falling back to Python.
   [(#1601)](https://github.com/PennyLaneAI/catalyst/pull/1601)
 
```

Comment:
````suggestion

For example, the following example will now be captured and executed properly with Autograph enabled:

```python
from catalyst import qjit

def updateList(x):
    return [x[0]+1, x[1]+2]

@qjit(autograph=True)
def fn(x):
    for i in range(4):
        x = updateList(x)
    return x
```

```pycon
>>> fn([1, 2])
[Array(5, dtype=int64), Array(10, dtype=int64)]
```

````

Reply:
@josh146

Reply:
looks good!